app/helpers/stripe/javascript_helper.rb

```diff
@@ -7,5 +7,20 @@ def stripe_javascript_tag(stripe_js_version = DEFAULT_STRIPE_JS_VERSION)
 
       render 'stripe/js', stripe_js_version: stripe_js_version
     end
+
+    def stripe_elements_tag(stripe_public_key: ENV["STRIPE_PUBLIC_KEY"], submit_path:, options: {})
+      default_options = {
+        label_text: "Credit or debit card",
```

Comment:
Cool. Should I add logic to the install generator that appends default values to en.yml?

Reply:
I'm undecided so I'll leave that to you. FYI though, i18n should work automatically since this is an engine:

> For locales, simply place the locale files in the config/locales directory, just like you would in an application.

https://guides.rubyonrails.org/engines.html#general-engine-configuration

Reply:
Didn't know about that, thanks. Pushed up a version that takes advantage of locales. I'll begin working on the documentation for this feature.